test/test_rosbag/test/test_bag.cpp.in

```diff
@@ -170,7 +170,7 @@ TEST_F(BagTest, bag_not_open_fails) {
         b.write("/test", ros::Time::now(), foo_);
         FAIL();
     }
-    catch (rosbag::BagIOException ex) {
+    catch (rosbag::BagIOException& ex) {
```

Comment:
`const`?

Reply:
Not that there is a difference - since the exception a temporary object anyway - but adding `const` for these is fine with me too.